apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1295,21 +1295,17 @@ get_onchain_env(MicroHeader) ->
                 key ->
                     {PrevHeader, PrevHeader};
                 micro ->
-                    case aec_chain:get_header(PrevKeyHash) of
-                        {ok, PrevKeyHeader} -> {PrevHeader, PrevKeyHeader};
-                        error               -> {PrevHeader, not_found}
-                    end
+                    {ok, PrevKeyHeader} = aec_chain:get_header(PrevKeyHash),
```

Comment:
Does this mean that you can crash the remote peer-process by sending an unconnected micro-block?

Reply:
Wasn't this exactly what I asked about in another PR? 😕